bin/setup.sh

```diff
@@ -29,10 +29,20 @@ if [[ ! -f $CONFIG_FILE ]]; then
 
   read BABY_BUDDY_API_KEY
 
+  echo -n "Enter CF Client Id: "
+
+  read CF_ACCESS_CLIENT_ID
+
+  echo -n "Enter CF Client Secret: "
+
+  read CF_ACCESS_CLIENT_SECRET
+
```

Comment:
Can we make the Cloudflare questions branching? E.g., ask first "Is the Baby Buddy instance protected by CloudFlare Zero Trust?". If yes, branch to the CF header questions and if no skip them (and don't add them to the config).